docs/First-Installation.md

```diff
@@ -58,24 +58,22 @@ The simplest way to get most of these components is through an all-in-one kit su
    * For Linux: [Raspberry Pi Imager](https://www.raspberrypi.org/software/), [Etcher](https://www.balena.io/etcher/), or the `dd` command
     * MacOS/Linux users can optionally extract the .img image from the downloaded .img.gz by using `gunzip` (macOS users can also simply double-click it)
-4.  Once the program's image/OS and SD card have been chosen, *write* it to the SD card and wait until the operation completes
-5.  Remove the MicroSD card from your computer, slide it back into the slot on your Raspberry Pi, and turn it on
+4. Once the program's image/OS and SD card have been chosen, *write* it to the SD card and wait until the operation completes
+5. Remove the MicroSD card from your computer, slide it back into the slot on your Raspberry Pi, and turn it on
 
 ## Configure Controllers
 
-On first boot your filesystem will be expanded automatically, you will then be welcomed with the following screen- this menu will configure your controls for both EmulationStation and RetroArch Emulators:
+On first boot you will be welcomed with the screen below. This menu will configure your controls for both EmulationStation and all RetroArch emulators:
 
 ![welcomescreen](https://cloud.githubusercontent.com/assets/10035308/9140482/cf42f25c-3cee-11e5-8f91-c1fc1c57175c.png)
 
-Hold down any button on your keyboard or gamepad and the name will appear at the bottom and then open up into a configuration menu:
-
-![welcomescreengamepadname](https://cloud.githubusercontent.com/assets/10035308/9140505/f5c19e38-3cee-11e5-965e-0e4e85ddaf56.png)
-
-Follow the onscreen instructions to configure your gamepad- if you run out of buttons just hold down a button to skip each unused button. **When you get to OK press the button you have configured as "A"**.
+Hold down any button on your controller or keyboard. While holding, its name will appear at the bottom of the window for a few seconds and then open up into a configuration menu:
 
 ![welcomescreengamepadconfigure](https://cloud.githubusercontent.com/assets/10035308/9140518/0263b9c8-3cef-11e5-922f-42f790f3be91.png)
 
-If you wish to configure more than one controller, you can do so from the start menu of EmulationStation. For more details on manual controller configurations see this page [Here](RetroArch-Configuration).
+Follow the onscreen instructions to configure your gamepad. If your controller doesn't have a button that you're being asked to define, just hold down any button to skip it.
+
+The [Hotkey](First-Installation#Hotkeys) button is recommended to be defined as the controller's *Home* button, which is usually centered on the controller ("X" on Xbox controllers, "P" on Playstation controllers, "Logitech" button, etc.). If a Home button isn't available, the *Select* button is a good alternative. Hotkeys are explained in more detail below.
```

Comment:
IMO this is too specific. We deal with keyboards, IPAC setups, all sorts of controllers, fightsticks and so on. I don't think it's right to refer to specific buttons on specific controllers.

Also,  I still prefer how it was laid out previously, with the intro to hotkeys and the table of combos all in the same section. Otherwise you're suggesting on how to bind a button the user doesn't know the function of, then linking to anchor below, but in a linear sequence of instructions. IMO it's better to have it all in the Hotkey section below, but perhaps slightly reworded to support dedicated buttons:

> The Hotkey Enable button enables you to press it in combination with another button to access functions such as saving states, loading states, and exiting back to EmulationStation. If there is no spare on your controller to use as a dedicated Hotkey Enable button, it is suggested to use the same button as Select.
>
> The following chart shows the default hotkey combinations. For example, if you chose Select as your Hotkey Enable button, that means you hold down Select while pressing the other button to execute the command.
> 
> (chart)

Reply:
I agree to an extent, but if new users are following along step by step, I recommend also moving the "When you get to "OK" at the end" bold line to below the Hotkeys section so it all flows better.

Reply:
Updated. How's that?

Reply:
> If your controller doesn't have a Home button, the *Select* button is a good alternative.

Again it's still specific to this concept of a 'Home' button - it's just any spare button on your input device, it doesn't matter what its designation is.

Reply:
> For example, the Home button

It's just an example? I'm confused.

Reply:
because the quoted section precludes the existence of input devices with a spare button but not 'home' button.

Reply:
I already have this:

> It is recommended to define an unused button or key as your Hotkey Enable button.

Here's an example:

> For example, the Home button on some controllers such as the center "X" on Xbox controllers.

Here's an alternative:

> If your controller doesn't have a Home button, the Select button is a good alternative.

I can't work out what you're trying to tell me. Should I not call it a "Home" button? Just, "Xbox button" or "Playstation button"?

Reply:
Because this alternative scenario is not just your example of a controller with or without home button, but for all controllers:

> If your controller doesn't have a Home button, the Select button is a good alternative.

should be 

> If your controller doesn't have an unused button, the Select button is a good alternative.

otherwise it’s like saying

> it’s best to wait for the correct signal before crossing the street. for example, in the UK this is a traffic light with a green man. If there’s no green man, don’t cross the street.

vs

> it’s best to wait for the correct signal before crossing the street. for example, in the UK this is a traffic light showing a green man. If there’s no signal, don’t cross the street.

otherwise people in the USA are going to be stuck on one side of the road...

Reply:
Oh, yeah I agree with that. 

[And yeah, those silly people in the USA](https://media.giphy.com/media/j2nATOAdRgYZq/giphy.gif)